Pod/Classes/OTNoop.m

```diff
@@ -24,7 +24,9 @@ @implementation OTNoopSpan
 - (id<OTTracer>)tracer { return g_defaultNoopTracer; }
 - (id<OTSpanContext>)context { return g_defaultNoopSpanContext; }
 - (void)setOperationName:(NSString*)operationName {}
-- (void)setTag:(NSString*)key value:(NSString*)value {}
+- (void)setTag:(NSString*)key value:(nonnull NSString *)value {}
```

Comment:
Good catch, implementation file doesn't need nonnull annotations, OTNoop conforms to OTSpan. I removed the extraneous annotations.